controllers/v1/projectAttributes.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * name : profile.js
+ * author : Mallanagouda R Biradar
+ * created-date : 13-Jun-2024
+ * Description :  profile Controller.
+ */
+
+// dependencies
+let projectAttributesHelper = require(MODULES_BASE_PATH + '/projectAttributes/helper')
```

Comment:
@praveenKDass  please  add api configurations for interface service

Reply:
Added @VISHNUDAS-tunerlabs